src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -1281,6 +1304,187 @@ internal BoundExpression BindCollectionBuilderCreate(
             return collectionCreation;
         }
 
+        private BoundExpression BindInterfaceTargetCollectionMethodGroup(
+            SyntaxNode syntax,
+            CollectionExpressionTypeKind collectionTypeKind,
+            NamedTypeSymbol targetType,
+            BindingDiagnosticBag diagnostics)
+        {
+            Debug.Assert(ContainingType is { });
+            const string methodName = "<signature>"; // Arbitrary name.
```

Comment:
Is this name going to be reflected in error messages? Or public APIs?

Reply:
The name is included in error messages. I don't think this method should be available from the `SemanticModel` though.